src/main/java/org/folio/rest/repository/OwnerRepository.java

```diff
@@ -15,9 +16,9 @@ public OwnerRepository(PostgresClient pgClient) {
     this.pgClient = pgClient;
   }
 
-  public Future<Owner> getById(String id) {
+  public Future<FeeFineNoticeContext> loadOwner(FeeFineNoticeContext context) {
     Future<Owner> future = Future.future();
-    pgClient.getById(OWNERS_TABLE, id, Owner.class, future);
-    return future;
+    pgClient.getById(OWNERS_TABLE, context.getFeefine().getOwnerId(), Owner.class, future);
```

Comment:
Probably extra check for null is needed here (same for other repository files)

Reply:
Done